wurst/util/Illusion.wurst

```diff
@@ -0,0 +1,102 @@
+package Illusion
```

Comment:
Why is this still in this PR?

Reply:
I told you, this package depends on damageModify. And since I'm working on my map too, I can't wait for it to be added after this pr is merged.

Reply:
I told you, this package depends on damageModify. And since I'm working on my map too, I can't wait for it to be added after this pr is merged.

Reply:
I told you, this package depends on damageModify. And since I'm working on my map too, I can't wait for it to be added after this pr is merged.

Reply:
I told you, this package depends on damageModify. And since I'm working on my map too, I can't wait for it to be added after this pr is merged.

Reply:
I told you, this package depends on damageModify. And since I'm working on my map too, I can't wait for it to be added after this pr is merged.

Reply:
But if it is required, then I'll cherrypick/revert this out of this PR.

Reply:
Revert it and use a second branch as a target for your own map.

When changes are made to `trokkin:DamageModify`, just make your map's branch rebase onto `trokkin:DamageModify`